pkg/action/cl001/ac010/error.go

```diff
@@ -5,6 +5,7 @@ import "github.com/giantswarm/microerror"
 // jobNotCompleted is an error indicating that the job in kiam test is not jet finished
 var jobNotCompleted = &microerror.Error{
 	Kind: "jobNotCompleted",
+	Desc: "AWS API call job for testing kiam is not completed",
```

Comment:
Food for thought. The error kind is about the what. The description you put here is about the what as well. Though its intention should be about the why and more background information.

Reply:
Ehm, so some ideas what to improve here?

Reply:
We could explain why it might not be completed and what to check for if this error happens.